src/components/AssetsClearFiltersButton/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import messages from './displayMessages';
+
+
+const AssetsClearFiltersButton = ({ clearFilters, courseDetails }) => (
```

Comment:
Nit: If this button clears both the filters and the search now, I wonder if we can find a better name for it. I can't really think of a good one, so if no one else is opposed, it's fine. Maybe, `AssetsShowAllFilesButton`? Although, like Farhanah mentioned, I could see that being confusing too.